Web Inspector: remove WebDatabase support from backend and frontend

burg opened the pull request:
#### 070fb7ec99ba0ba82848fa2c248646e4bd728ea6
<pre>
Web Inspector: remove WebDatabase support from backend and frontend
<a href="https://rdar.apple.com/143596276">rdar://143596276</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=286634">https://bugs.webkit.org/show_bug.cgi?id=286634</a>

Reviewed by Devin Rousso.

WebSQL has not shipped in several years. In preparation for its full removal
from WebKit, start by ripping out the related code in Web Inspector.

This domain never had any layout tests associated with it, so there are no deleted tests.

* LayoutTests/inspector/runtime/CommandLineAPI-inspect.html:
* Source/JavaScriptCore/CMakeLists.txt:
* Source/JavaScriptCore/DerivedSources-input.xcfilelist:
* Source/JavaScriptCore/DerivedSources.make:
* Source/JavaScriptCore/inspector/protocol/Database.json: Removed.
* Source/WebCore/Modules/webdatabase/DatabaseManager.cpp:
(WebCore::DatabaseManager::openDatabase):
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedLocalVarsCheckerExpectations:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/inspector/CommandLineAPIHost.cpp:
(WebCore::CommandLineAPIHost::databaseId): Deleted.
* Source/WebCore/inspector/CommandLineAPIHost.h:
* Source/WebCore/inspector/CommandLineAPIHost.idl:
* Source/WebCore/inspector/InspectorController.cpp:
(WebCore::InspectorController::createLazyAgents):
* Source/WebCore/inspector/InspectorDatabaseResource.cpp: Removed.
* Source/WebCore/inspector/InspectorDatabaseResource.h: Removed.
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
(WebCore::InspectorInstrumentation::didCommitLoadImpl):
(WebCore::InspectorInstrumentation::didOpenDatabaseImpl): Deleted.
* Source/WebCore/inspector/InspectorInstrumentation.h:
(WebCore::InspectorInstrumentation::didOpenDatabase): Deleted.
* Source/WebCore/inspector/InstrumentingAgents.h:
* Source/WebCore/inspector/agents/InspectorDatabaseAgent.cpp: Removed.
* Source/WebCore/inspector/agents/InspectorDatabaseAgent.h: Removed.
* Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js:
* Source/WebInspectorUI/UserInterface/Base/Main.js:
(WI.loaded):
* Source/WebInspectorUI/UserInterface/Controllers/DatabaseManager.js: Removed.
* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Models/DatabaseObject.js: Removed.
* Source/WebInspectorUI/UserInterface/Models/DatabaseTableObject.js: Removed.
* Source/WebInspectorUI/UserInterface/Models/NativeFunctionParameters.js:
* Source/WebInspectorUI/UserInterface/Protocol/DatabaseObserver.js: Removed.
* Source/WebInspectorUI/UserInterface/Protocol/InspectorObserver.js:
(WI.InspectorObserver.prototype.inspect):
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/12.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/12.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/13.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/13.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/14.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/14.5/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/15.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/15.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/16.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/16.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/18.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/18.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/12.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/12.3/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/13.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/13.3/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/15.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/15.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Target.js:
(WI.Target.prototype.get DOMStorageAgent):
(WI.Target.prototype.get DatabaseAgent): Deleted.
* Source/WebInspectorUI/UserInterface/Test.html:
* Source/WebInspectorUI/UserInterface/Test/Test.js:
(WI.loaded):
(WI.contentLoaded):
* Source/WebInspectorUI/UserInterface/Views/ContentView.js:
(WI.ContentView.createFromRepresentedObject):
(WI.ContentView.isViewable):
* Source/WebInspectorUI/UserInterface/Views/DatabaseContentView.css: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseContentView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseHostTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableContentView.css: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableContentView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQueryErrorView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQuerySuccessView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQueryViewBase.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js:
(WI.NavigationSidebarPanel.prototype._isTreeElementWithoutRepresentedObject):
* Source/WebInspectorUI/UserInterface/Views/StorageSidebarPanel.js:
(WI.StorageSidebarPanel):
(WI.StorageSidebarPanel.prototype.closed):
(WI.StorageSidebarPanel._treeSelectionDidChange):
(WI.StorageSidebarPanel.prototype._databaseWasAdded): Deleted.
(WI.StorageSidebarPanel.prototype._addDatabase): Deleted.
(WI.StorageSidebarPanel.prototype._databaseWasInspected): Deleted.
(WI.StorageSidebarPanel.prototype._databaseCleared): Deleted.
* Source/WebInspectorUI/UserInterface/Views/StorageTabContentView.js:
(WI.StorageTabContentView):
(WI.StorageTabContentView.isTabAllowed):
(WI.StorageTabContentView.prototype.canShowRepresentedObject):
(WI.StorageTabContentView.prototype.closed):
* Source/WebInspectorUI/Versions/Inspector-iOS-12.2.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-13.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-13.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-14.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-14.5.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-15.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-15.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-16.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-16.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.2.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-18.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-18.2.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-12.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-12.3.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-13.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-13.3.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.2.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.4.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-15.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-15.2.json:

Canonical link: <a href="https://commits.webkit.org/289854@main">https://commits.webkit.org/289854@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4e71f5e802120f64df140d4b529272a170098b89

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/88153 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/7671 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/42605 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/93107 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/38903 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/90204 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/8050 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/15847 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/68028 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/25751 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/91155 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/6159 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/79768 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/48396 "Found 1 new API test failure: /TestJSC:/jsc/options (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/5933 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/34180 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/38011 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/80951 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/76319 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/35061 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/94950 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/86929 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/15321 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/11244 "Found 3 new test failures: fast/dom/crash-with-bad-url.html imported/w3c/web-platform-tests/screen-orientation/lock-unlock-check.html imported/w3c/web-platform-tests/screen-orientation/unlock.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/76882 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/15577 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/75624 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/76126 "Found 1 new API test failure: /TestJSC:/jsc/options (failure)") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/18733 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/20516 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/18951 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/8350 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/15343 "Built successfully") | [❌ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/20640 "Found 1 new failure in inspector/agents/worker/WorkerNetworkAgent.h and found 2 fixed files: inspector/agents/worker/WorkerDebuggerAgent.h, inspector/agents/InspectorDatabaseAgent.cpp") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/109422 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/15083 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/26311 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/18529 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/16865 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/DatabaseContentView.js:
```diff
@@ -1,121 +0,0 @@
-
-        this.element.classList.add("storage-view", "query", "monospace");
-
-        this._prompt = new WI.ConsolePrompt(this, "text/x-sql");
```
i think with this being removed we can simplify `WI.ConsolePrompt` a little bit (e.g. remove `mimeType` parameter, remove `consolePromptCompletionsNeeded` delegate method, delete the `Source/WebInspectorUI/UserInterface/External/CodeMirror/sql.js` file, etc.)

burg replied:
In the interest of getting this done, that needs to be in a separate patch.

dcrousso commented on Source/WebInspectorUI/UserInterface/Protocol/InspectorObserver.js:
```diff
@@ -58,8 +58,6 @@ WI.InspectorObserver = class InspectorObserver extends InspectorBackend.Dispatch
 
-        if (hints.databaseId)
-            WI.databaseManager.inspectDatabase(hints.databaseId);
         else if (hints.domStorageId)
```
oops

dcrousso commented on Source/WebInspectorUI/UserInterface/Base/Main.js:
```diff
@@ -1412,7 +1408,6 @@ WI.tabContentViewClassForRepresentedObject = function(representedObject)
         return WI.SourcesTabContentView;
 
     if (representedObject instanceof WI.DOMStorageObject || representedObject instanceof WI.CookieStorageObject ||
-        representedObject instanceof WI.DatabaseTableObject || representedObject instanceof WI.DatabaseObject ||
```
there's a few other values on `WI.ContentViewCookieType` that probably should be removed

dcrousso commented on Source/WebInspectorUI/UserInterface/Base/Main.js:
```diff
@@ -147,7 +144,6 @@ WI.loaded = function()
     WI.domManager.addEventListener(WI.DOMManager.Event.InspectModeStateChanged, WI._inspectModeStateChanged, WI);
     WI.domManager.addEventListener(WI.DOMManager.Event.DOMNodeWasInspected, WI._domNodeWasInspected, WI);
     WI.domStorageManager.addEventListener(WI.DOMStorageManager.Event.DOMStorageObjectWasInspected, WI._domStorageWasInspected, WI);
-    WI.databaseManager.addEventListener(WI.DatabaseManager.Event.DatabaseWasInspected, WI._databaseWasInspected, WI);
```
please also remove `WI._databaseWasInspected`

dcrousso commented on Source/WebInspectorUI/UserInterface/Test/Test.js:
```diff
@@ -36,7 +36,6 @@ WI.loaded = function()
     InspectorBackend.registerConsoleDispatcher(WI.ConsoleObserver);
     InspectorBackend.registerDOMDispatcher(WI.DOMObserver);
     InspectorBackend.registerDOMStorageDispatcher(WI.DOMStorageObserver);
-    InspectorBackend.registerDatabaseDispatcher(WI.DatabaseObserver);
```
please also remove this below
```js
makeAgentGetter("Database");
```

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/StorageTabContentView.js:
```diff
@@ -75,8 +74,6 @@ WI.StorageTabContentView = class StorageTabContentView extends WI.ContentBrowser
         return representedObject instanceof WI.ApplicationCacheFrame
             || representedObject instanceof WI.CookieStorageObject
             || representedObject instanceof WI.DOMStorageObject
-            || representedObject instanceof WI.DatabaseObject
-            || representedObject instanceof WI.DatabaseTableObject
```
there's another check above we should remove
```js
InspectorBackend.hasDomain("Database")
```

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/DatabaseUserQuerySuccessView.js:
```diff
@@ -1,60 +0,0 @@
-            this._dataGrid.updateLayoutIfNeeded();
-        } else {
-            this.resultElement.classList.add("no-results");
-            this.resultElement.textContent = WI.UIString("Query returned no results.");
```
please dont forget to update `Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js` :)

dcrousso commented on Source/WebInspectorUI/UserInterface/Protocol/Target.js:
```diff
@@ -140,7 +140,6 @@ WI.Target = class Target extends WI.Object
     get DOMAgent() { return this._agents.DOM; }
     get DOMDebuggerAgent() { return this._agents.DOMDebugger; }
     get DOMStorageAgent() { return this._agents.DOMStorage; }
-    get DatabaseAgent() { return this._agents.Database; }
```
im slightly hesitant about removing this from the UI as it means we'll be dropping support when the backend technically could still work, but seeing as how this all has been disabled for so long i think it's fine

if we are gonna remove the frontend bits, however, we should manually edit all of the `Source/WebInspectorUI/UserInterface/Protocol/Legacy/*/*/InspectorBackendCommands.js` to not have the `Database` domain as otherwise we'll still create the `WI.DatabaseObserver` class, `DatabaseAgent` class, etc. which is just wasteful

dcrousso reviewed (approved):
awesome! a few things to fix but overall very excited to get rid of this code :)

burg committed:
Web Inspector: remove WebDatabase support from backend and frontend
rdar://143596276
https://bugs.webkit.org/show_bug.cgi?id=286634

Reviewed by Devin Rousso.

WebSQL has not shipped in several years. In preparation for its full removal
from WebKit, start by ripping out the related code in Web Inspector.

This domain never had any layout tests associated with it, so there are no deleted tests.

* LayoutTests/inspector/runtime/CommandLineAPI-inspect.html:
* Source/JavaScriptCore/CMakeLists.txt:
* Source/JavaScriptCore/DerivedSources-input.xcfilelist:
* Source/JavaScriptCore/DerivedSources.make:
* Source/JavaScriptCore/inspector/protocol/Database.json: Removed.
* Source/WebCore/Modules/webdatabase/DatabaseManager.cpp:
(WebCore::DatabaseManager::openDatabase):
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedLocalVarsCheckerExpectations:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/inspector/CommandLineAPIHost.cpp:
(WebCore::CommandLineAPIHost::databaseId): Deleted.
* Source/WebCore/inspector/CommandLineAPIHost.h:
* Source/WebCore/inspector/CommandLineAPIHost.idl:
* Source/WebCore/inspector/InspectorController.cpp:
(WebCore::InspectorController::createLazyAgents):
* Source/WebCore/inspector/InspectorDatabaseResource.cpp: Removed.
* Source/WebCore/inspector/InspectorDatabaseResource.h: Removed.
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
(WebCore::InspectorInstrumentation::didCommitLoadImpl):
(WebCore::InspectorInstrumentation::didOpenDatabaseImpl): Deleted.
* Source/WebCore/inspector/InspectorInstrumentation.h:
(WebCore::InspectorInstrumentation::didOpenDatabase): Deleted.
* Source/WebCore/inspector/InstrumentingAgents.h:
* Source/WebCore/inspector/agents/InspectorDatabaseAgent.cpp: Removed.
* Source/WebCore/inspector/agents/InspectorDatabaseAgent.h: Removed.
* Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js:
* Source/WebInspectorUI/UserInterface/Base/Main.js:
(WI.loaded):
* Source/WebInspectorUI/UserInterface/Controllers/DatabaseManager.js: Removed.
* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Models/DatabaseObject.js: Removed.
* Source/WebInspectorUI/UserInterface/Models/DatabaseTableObject.js: Removed.
* Source/WebInspectorUI/UserInterface/Models/NativeFunctionParameters.js:
* Source/WebInspectorUI/UserInterface/Protocol/DatabaseObserver.js: Removed.
* Source/WebInspectorUI/UserInterface/Protocol/InspectorObserver.js:
(WI.InspectorObserver.prototype.inspect):
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/12.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/12.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/13.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/13.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/14.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/14.5/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/15.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/15.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/16.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/16.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/17.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/18.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/18.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/12.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/12.3/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/13.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/13.3/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/14.4/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/15.0/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Legacy/macOS/15.2/InspectorBackendCommands.js:
* Source/WebInspectorUI/UserInterface/Protocol/Target.js:
(WI.Target.prototype.get DOMStorageAgent):
(WI.Target.prototype.get DatabaseAgent): Deleted.
* Source/WebInspectorUI/UserInterface/Test.html:
* Source/WebInspectorUI/UserInterface/Test/Test.js:
(WI.loaded):
(WI.contentLoaded):
* Source/WebInspectorUI/UserInterface/Views/ContentView.js:
(WI.ContentView.createFromRepresentedObject):
(WI.ContentView.isViewable):
* Source/WebInspectorUI/UserInterface/Views/DatabaseContentView.css: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseContentView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseHostTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableContentView.css: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableContentView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTableTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseTreeElement.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQueryErrorView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQuerySuccessView.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/DatabaseUserQueryViewBase.js: Removed.
* Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js:
(WI.NavigationSidebarPanel.prototype._isTreeElementWithoutRepresentedObject):
* Source/WebInspectorUI/UserInterface/Views/StorageSidebarPanel.js:
(WI.StorageSidebarPanel):
(WI.StorageSidebarPanel.prototype.closed):
(WI.StorageSidebarPanel._treeSelectionDidChange):
(WI.StorageSidebarPanel.prototype._databaseWasAdded): Deleted.
(WI.StorageSidebarPanel.prototype._addDatabase): Deleted.
(WI.StorageSidebarPanel.prototype._databaseWasInspected): Deleted.
(WI.StorageSidebarPanel.prototype._databaseCleared): Deleted.
* Source/WebInspectorUI/UserInterface/Views/StorageTabContentView.js:
(WI.StorageTabContentView):
(WI.StorageTabContentView.isTabAllowed):
(WI.StorageTabContentView.prototype.canShowRepresentedObject):
(WI.StorageTabContentView.prototype.closed):
* Source/WebInspectorUI/Versions/Inspector-iOS-12.2.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-13.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-13.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-14.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-14.5.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-15.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-15.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-16.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-16.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.2.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-17.4.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-18.0.json:
* Source/WebInspectorUI/Versions/Inspector-iOS-18.2.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-12.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-12.3.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-13.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-13.3.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.2.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-14.4.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-15.0.json:
* Source/WebInspectorUI/Versions/Inspector-macOS-15.2.json:

Canonical link: https://commits.webkit.org/289854@main
